src/components/DataCollector/createEvent.js

```diff
@@ -10,7 +10,8 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import createPayloadItemMerger from "./createPayloadItemMerger";
+// TODO move this dependency to utils
+import { createMerger } from "../../utils";
```

Comment:
It is now used by the event object, so it is more than just a "payload" item merger.  Also, I moved it to the utils because it is now used by core, and by the data collector (via the event object).